archetypes/helidon/src/main/archetype/se/custom/observability.xml

```diff
@@ -37,7 +43,7 @@ tracing:
     log-spans: false      # JAEGER_REPORTER_LOG_SPANS
     max-queue-size: 42    # JAEGER_REPORTER_MAX_QUEUE_SIZE
     flush-interval-ms: 10001 # JAEGER_REPORTER_FLUSH_INTERVAL
-    sampler-type: "remote"# JAEGER_SAMPLER_TYPE (https://www.jaegertracing.io/docs/latest/sampling/#client-sampling-configuration)
+    sampler-type: "constant"# JAEGER_SAMPLER_TYPE (https://www.jaegertracing.io/docs/latest/sampling/#client-sampling-configuration)
```

Comment:
Please go through the linked documentation (from issue). Some of the options are no longer available (this was just an example - there are others that are not supported, such as `sampler-manager` etc.)